source/_integrations/plugwise.markdown

```
@@ -224,45 +194,219 @@ action: `climate.set_preset_mode`
+
+### Select
+
+[Select](/integrations/select) allows for changing a thermostat schedule. If you have an Adam you can also select the regulation mode.
+
+Schedules can be created using the Plugwise App or the web-interface.
+
+#### Change climate schedule
+
+action: `select.select_option`
+
+Assuming you have something to indicate your current day of night shift based on a calendar you can automatically change the active schedule. The automation example below assumes an (`input_select`)[/integrations/input_select/] helper is present, e.g., containing something like `Day shift`, `Night shift` and `Weekend shift`). Assuming you have created the appropriate schedules using your Plugwise App, your automation for night shifts could look like:
```

Markdownlint (0.37.0)

(`input_select`)[/integrations/input_select/]
Reversed link syntax

(MD011, no-reversed-links)